### ISSUE TITLE
Error at installation

### ISSUE THREAD

**Issue by Thomseeen** [aleho/gnome-shell-volume-mixer#104]
Installing from Gnome Extensions prompts ERROR. After a system restart I could open the extensions settings to get prompted with:
```
The settings of extension shell-volume-mixer@derhofbauer.at had an error:

GLib.SpawnError: Failed to execute child process “python” (No such file or directory)

Stack trace:

getCards@/home/thomas/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/lib/utils/utils.js:91:29
_initCards@/home/thomas/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:108:21
buildWidget@/home/thomas/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:56:9
buildPrefsWidget@/home/thomas/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:490:12
_selectExtension@resource:///org/gnome/shell/extensionPrefs/main.js:75:22
_onCommandLine@resource:///org/gnome/shell/extensionPrefs/main.js:320:17
main@resource:///org/gnome/shell/extensionPrefs/main.js:635:5
@<main>:1:43
```

I don't know whether that helps but python3 is installed.

**Comment by aleho** [comment]
What does calling `/usr/bin/env python` result in?

**Comment by Thomseeen** [comment]
A few days, a few restarts and an `apt update` later, everything works without me doing anything special. I just checked back on it and it works as intended. Weird one...
Now, as it is working, Python 2.7 is opening up after said command.
For me this issue would be closed now...

**Comment by aleho** [comment]
Actually this might be related to the switch from Python2 to 3 and dropping of the former.

Could reproduce here.

**Comment by janekbt** [comment]
Hi everyone, I am experiencing the same error. This is my error message:

`GLib.SpawnError: Failed to execute child process “python” (No such file or directory)

Stack trace:
  getCards@/home/janek/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/utils.js:90:34
  _initCards@/home/janek/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:116:27
  wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
  buildWidget@/home/janek/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:60:14
  wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
  buildPrefsWidget@/home/janek/.local/share/gnome-shell/extensions/shell-volume-mixer@derhofbauer.at/prefs.js:505:24
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:13
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:175:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:222:19
  start@resource:///org/gnome/gjs/modules/script/package.js:206:5
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17`

Calling `/usr/bin/env python` results in `/usr/bin/env: ‘python’: No such file or directory` but python3 is installed on my system. Installing the python-is-python3 package resolves that error, so that instead of the error message the settings window pops-up, but the extension web page still shows an error button and the extension doesn't show up in the top right menu.

I am using Ubuntu 20.04 with all the updates installed while using the vanilla gnome 3 desktop. If any more info is needed to solve that problem I'll gladly provide it, and if it is my fault that the extensions doesn't work, which is probably the case, then I am already sorry for wasting your time! :)

**Comment by aleho** [comment]
Ah, this is stupid. Why isn't there a python bin if python3 is installed? I don't get distributions 🤷🏼.

Anyway, I think I can easily fix this (by going back to a previous fix).

**Comment by aleho** [comment]
No, actually spoke too soon, this fix is still there.

@janekbt You're using an outdated version I cannot support any more. Please consider updating (the fix was introduced with 3.36.6).
